src/main/resources/db/migration/V20191004032400__create_table_locations.sql

```diff
@@ -5,7 +5,8 @@ CREATE TABLE locations
     postal_code    VARCHAR2(12),
     city           VARCHAR2(30),
     state_province VARCHAR2(25),
-    country_id     CHAR(2) NOT NULL
+    country_id     CHAR(2) NOT NULL,
+    department_amount NUMBER(3)
```

Comment:
Please create a separated migration to add this new column.

Reply:
created